cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -130,23 +131,17 @@ def _set_schema_def(self):
         """
         Sets schema dictionary from using information in adata. If there are any errors, it adds them to self.errors
         """
-        # Check if schema version slot is present
-        if "schema_version" not in self.adata.uns:
-            self.errors.append("adata has no schema definition in 'adata.uns'. Validation cannot be performed.")
-        else:
-            # Check if schema version is supported
-            version = self.adata.uns["schema_version"]
-            path = schema.get_schema_file_path(version)
-
-            if not os.path.isfile(path):
-                supported_versions = schema.get_schema_versions_supported()
-                self.errors.append(
-                    f"Schema version '{version}' is not supported. Current supported versions: '{supported_versions}'. "
-                    f"Validation cannot be performed."
-                )
-            else:
-                if not self.schema_def:
-                    self.schema_def = schema.get_schema_definition(version)
+        version = self.adata.uns.get("schema_version")
+        supported_versions = schema.get_schema_versions_supported()
+        supported_versions.sort()
+        if version and version != supported_versions[-1]:
```

Comment:
should we update this to only check against major version, as suggested in stand-up?

Reply:
Adding the complexity of comparing strictly the major version does not give us any added value. In practice we only ever have one schema that the CLI supports. So this should only ever have the latest schema.

Reply:
From #440 we will forcefully validate against the lasted scheme version regardless of the what the `schema_version` in the h5ad.

Reply:
True, but it means we'll always print this warning from the first minor schema update onwards. Since the yaml name is 3.0.0 and a new one is only produced for major version updates.

Reply:
It would also produce a confusing warning message, since it'd read:
`"Schema version '3.1.0' is not supported. Current supported versions: [3.0.0]. "`

Reply:
Maybe the change should be in [how we derive the supported versions](https://github.com/chanzuckerberg/single-cell-curation/blob/d18856cc00be4ef078caa379a413403a16b99d04/cellxgene_schema_cli/cellxgene_schema/schema.py#L20)